happy/HappyInternet.py

```diff
@@ -266,7 +267,7 @@ def __nmconf(self):
             if tries <= 0:
                 emsg = "Failed to setup host interface %s with nmcli. Internet may not working." % \
                     (self.internet_node_end)
-                self.logger.warning("[localhost] HappyInternet: %s" % (emsg))
+                self.logger.debug("[localhost] HappyInternet: %s" % (emsg))
```

Comment:
What happens when there is a real problem here? The user will not get the warning, correct?

Rather than burying both spurious and real warnings, why don't we ensure the spurious warning doesn't happen?

Reply:
I am thinking to change this back to logger.warning
I will discuss with @robszewczyk for this one after he come back 2 weeks later, and will address this one in a separate PR.